### PR TITLE
[CLI] cobra lots of things

### PR DESCRIPTION
**Pull request by ajm188**
## Description

Migrates the following binaries to cobra:
- `vtaclcheck`
- `vtbackup`
- `vtbench` (keep this?)
- `vtclient`
- `vtcombo`

## Related Issue(s)

#13918 

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on the CI
-   [x] Documentation was added or is not required: https://github.com/vitessio/website/pull/1587

### CODE REVIEW

File: go/flags/endtoend/vtbackup.txt [hunk 1]
```diff
@@ -1,4 +1,47 @@
...
+    mysqlctld in the target shard to be backed up.
+	- Provisioning as much disk space for vtbackup as would be given to vttablet.
+    The data directory MUST be empty at startup. Do NOT reuse a persistent disk.
+	- Running vtbackup periodically for each shard, for each backup storage location.
+	- Ensuring that at most one instance runs at a time for a given pair of shard
+    and backup storage location.
+	- Retrying vtbackup if it fails.
+	- Alerting human operators if the failure is persistent.
+
+The process vtbackup follows to take a new backup is as follows:
```
Comment (deepthi) [inline go/flags/endtoend/vtbackup.txt:18]:
```suggestion
The process vtbackup follows to take a new backup has the following steps:
```

File: go/flags/endtoend/vtclient.txt [hunk 2]
```diff
@@ -0,0 +1,54 @@
+vtclient connects to a vtgate server using the standard go driver API.
+
+Version 3 of the API is used, we do not send any hint to the server.
```
Comment (deepthi) [inline go/flags/endtoend/vtclient.txt:3]: This line can be removed.

File: go/flags/endtoend/vtcombo.txt [hunk 3]
```diff
@@ -0,0 +1,432 @@
+vtcombo is a single binary containing several vitess components.
+
+In particular, it contains:
+- A ZK topology server based on an in-memory map.
```
Comment (deepthi) [inline go/flags/endtoend/vtcombo.txt:4]:
```suggestion
- A topology server based on an in-memory map.
```

**Review by deepthi — APPROVED** [review]
A few minor changes are needed, otherwise LGTM.

File: go/flags/endtoend/vtbackup.txt (hunk 1 above)
Comment (ajm188) [reply go/flags/endtoend/vtbackup.txt:18]: cant commit this because it has to go in the actual help, but copying it over there

**Review by frouioui — APPROVED** [review]

**Pull request merged**